lib/puppet/file_serving/content.rb

```diff
@@ -39,7 +39,7 @@ def content
     @content
   end
 
-  def to_raw
+  def to_binary
     File.new(full_path, "rb")
```

Comment:
we should probably change this to `Puppet::FileSystem::binread` like the file_bucket/file above.